projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -26,122 +26,124 @@
 </div>
 <ng-container *ngIf="record">
   <!-- HEADER -->
-  <header class="col-10 mt-3 mb-3">
-    <!-- TYPE -->
-    <ng-container *ngIf="record.metadata.type">
-      <span id="doc-type">{{ record.metadata.type | translate | titlecase }}</span>
-    </ng-container>
+  <header class="col-10">
+    <div class="row mt-3 mb-3">
+      <div class="col-2">
+        <shared-thumbnail [record]="record"></shared-thumbnail>
```

Comment:
Added the class parameter to be more flexible. The CSS definition stays in the main application.

Reply:
I do not understand this comment. Can you explain again?